app/pages/complete-order/template.njk

```diff
@@ -0,0 +1,37 @@
+{% extends 'includes/layout.njk' %}
+{% from 'components/back-link/macro.njk' import backLink %}
+{% from 'components/bc-button/macro.njk' import bcButton %}
+{% from 'components/bc-error-summary/macro.njk' import bcErrorSummary %}
+
+{% block body %}
+  <div class="nhsuk-width-container" data-test-id="complete-order-page">
+    <div data-test-id="go-back-link">
+      {{ backLink({
+        "href": backLinkHref,
+        "text": backLinkText
+      }) }}
+    </div>
+
+    <div class="nhsuk-grid-row">
+      <div class="nhsuk-grid-column-two-thirds">
+        {% if (errors.length > 0) %}
+          {{ bcErrorSummary({
+            errors: errors
+          }) }}
+        {% endif %}
```

Comment:
Are we ever going to have any errors to show on this page?

Reply:
no we won't will remove